vpr/src/route/rr_graph.cpp

```diff
@@ -2116,6 +2123,9 @@ static std::function<void(t_chan_width*)> alloc_and_load_rr_graph(RRGraphBuilder
     the edges are not remapped yet.*/
     bool switches_remapped = false;
 
+    // Define verbosity locally using router_opts
+    const int verbosity = route_verbosity;
```

Comment:
This line and comment are now useless. Please remove them and replace the uses of `verbosity` with `route_verbosity`.